remote_rest/client.py

```diff
@@ -0,0 +1,403 @@
+import argparse
+import json
+import logging
+import os
+import sys
+from typing import Any, Dict, List, Optional
+
```

Comment:
this should use our remote_client object. and we will put that executable to the attackmate-api-repo